commit ec44be54b85e45376ebd3473c931cac1843e70f2
Author: pranomostro <pranomostro@posteo.net>
Date:   Sun Feb 11 01:29:45 2018 +0100

    Updated nodes, fixed libraries. No solution for the hanging-up crash yet, though. :(

diff --git a/config.mk b/config.mk
index 470ee7e..0920710 100644
--- a/config.mk
+++ b/config.mk
@@ -8,6 +8,6 @@ MANPREFIX = $(PREFIX)/share/man
 CC = cc
 LD = $(CC)
 CPPFLAGS = -DVERSION=\"${VERSION}\"
-CFLAGS   = -I/usr/local/include -Wall -Wunused $(CPPFLAGS)
-LDFLAGS  = -L/usr/local/lib
-LDLIBS   = -ltoxcore -ltoxav -ltoxencryptsave -lsodium -lopus -lvpx -lm -lpthread
+CFLAGS   = -I/usr/local/include -Wall -Wunused $(CPPFLAGS) -g
+LDFLAGS  = -L/usr/local/lib64 -g
+LDLIBS   = -ltoxcore -lsodium -lopus -lvpx -lm -lpthread
diff --git a/nodes.h b/nodes.h
index 979a306..e659681 100644
--- a/nodes.h
+++ b/nodes.h
@@ -1,6 +1,6 @@
 static struct node nodes[] = {
 	{
-		.addr4 = "biribiri.org",
+		.addr4 = "node.tox.biribiri.org",
 		.addr6 = NULL,
 		.udp_port = 33445,
 		.tcp_port = 3389,
@@ -24,7 +24,7 @@ static struct node nodes[] = {
 		.addr4 = "205.185.116.116",
 		.addr6 = NULL,
 		.udp_port = 33445,
-		.tcp_port = 33445,
+		.tcp_port = 3389,
 		.idstr = "A179B09749AC826FF01F37A9613F6B57118AE014D4196A0E1105A98F93A54702"
 	},
 	{
@@ -34,18 +34,25 @@ static struct node nodes[] = {
 		.tcp_port = 3389,
 		.idstr = "1D5A5F2F5D6233058BF0259B09622FB40B482E4FA0931EB8FD3AB8E7BF7DAF6F"
 	},
+	{
+		.addr4 = "85.172.30.117",
+		.addr6 = NULL,
+		.udp_port = 33445,
+		.tcp_port = 33445,
+		.idstr = "8E7D0B859922EF569298B4D261A8CCB5FEA14FB91ED412A7603A585A25698832"
+	},
 	{
 		.addr4 = "194.249.212.109",
 		.addr6 = "2001:1470:fbfe::109",
 		.udp_port = 33445,
-		.tcp_port = 3389,
+		.tcp_port = 33445,
 		.idstr = "3CEE1F054081E7A011234883BC4FC39F661A55B73637A5AC293DDF1251D9432B"
 	},
 	{
 		.addr4 = "185.25.116.107",
 		.addr6 = "2a00:7a60:0:746b::3",
 		.udp_port = 33445,
-		.tcp_port = 33445,
+		.tcp_port = 3389,
 		.idstr = "DA4E4ED4B697F2E9B000EEFE3A34B554ACD3F45F5C96EAEA2516DD7FF9AF7B43"
 	},
 	{
@@ -59,21 +66,21 @@ static struct node nodes[] = {
 		.addr4 = "217.182.143.254",
 		.addr6 = "2001:41d0:302:1000::e111",
 		.udp_port = 2306,
-		.tcp_port = 2306,
+		.tcp_port = 443,
 		.idstr = "7AED21F94D82B05774F697B209628CD5A9AD17E0C073D9329076A4C28ED28147"
 	},
 	{
 		.addr4 = "104.223.122.15",
 		.addr6 = "2607:ff48:aa81:800::35eb:1",
 		.udp_port = 33445,
-		.tcp_port = 3389,
+		.tcp_port = 33445,
 		.idstr = "0FB96EEBFB1650DDB52E70CF773DDFCABE25A95CC3BB50FC251082E4B63EF82A"
 	},
 	{
 		.addr4 = "tox.verdict.gg",
 		.addr6 = NULL,
 		.udp_port = 33445,
-		.tcp_port = 3389,
+		.tcp_port = 33445,
 		.idstr = "1C5293AEF2114717547B39DA8EA6F1E331E5E358B35F9B6B5F19317911C5F976"
 	},
 	{
@@ -83,18 +90,11 @@ static struct node nodes[] = {
 		.tcp_port = 1813,
 		.idstr = "53737F6D47FA6BD2808F378E339AF45BF86F39B64E79D6D491C53A1D522E7039"
 	},
-	{
-		.addr4 = "104.233.104.126",
-		.addr6 = NULL,
-		.udp_port = 33445,
-		.tcp_port = 33445,
-		.idstr = "EDEE8F2E839A57820DE3DA4156D88350E53D4161447068A3457EE8F59F362414"
-	},
 	{
 		.addr4 = "51.254.84.212",
 		.addr6 = "2001:41d0:a:1a3b::18",
 		.udp_port = 33445,
-		.tcp_port = 3389,
+		.tcp_port = 33445,
 		.idstr = "AEC204B9A4501412D5F0BB67D9C81B5DB3EE6ADA64122D32A3E9B093D544327D"
 	},
 	{
@@ -104,20 +104,6 @@ static struct node nodes[] = {
 		.tcp_port = 33445,
 		.idstr = "2D320F971EF2CA18004416C2AAE7BA52BF7949DB34EA8E2E21AF67BD367BE211"
 	},
-	{
-		.addr4 = "185.58.206.164",
-		.addr6 = "2a02:f680:1:1100::3313",
-		.udp_port = 33445,
-		.tcp_port = 33445,
-		.idstr = "24156472041E5F220D1FA11D9DF32F7AD697D59845701CDD7BE7D1785EB9DB39"
-	},
-	{
-		.addr4 = "195.93.190.6",
-		.addr6 = "2a01:d0:ffff:a8a::2",
-		.udp_port = 33445,
-		.tcp_port = 33445,
-		.idstr = "FB4CE0DDEFEED45F26917053E5D24BDDA0FA0A3D83A672A9DA2375928B37023D"
-	},
 	{
 		.addr4 = "tox.uplinklabs.net",
 		.addr6 = "tox.uplinklabs.net",
@@ -129,14 +115,14 @@ static struct node nodes[] = {
 		.addr4 = "toxnode.nek0.net",
 		.addr6 = "toxnode.nek0.net",
 		.udp_port = 33445,
-		.tcp_port = 3389,
+		.tcp_port = 33445,
 		.idstr = "20965721D32CE50C3E837DD75B33908B33037E6225110BFF209277AEAF3F9639"
 	},
 	{
 		.addr4 = "95.215.44.78",
 		.addr6 = "2a02:7aa0:1619::c6fe:d0cb",
 		.udp_port = 33445,
-		.tcp_port = 3389,
+		.tcp_port = 33445,
 		.idstr = "672DBE27B4ADB9D5FB105A6BB648B2F8FDB89B3323486A7A21968316E012023C"
 	},
 	{
@@ -150,7 +136,7 @@ static struct node nodes[] = {
 		.addr4 = "sorunome.de",
 		.addr6 = "sorunome.de",
 		.udp_port = 33445,
-		.tcp_port = 3389,
+		.tcp_port = 33445,
 		.idstr = "02807CF4F8BB8FB390CC3794BDF1E8449E9A8392C5D3F2200019DA9F1E812E46"
 	},
 	{
@@ -160,25 +146,18 @@ static struct node nodes[] = {
 		.tcp_port = 33445,
 		.idstr = "E59A0E71ADA20D35BD1B0957059D7EF7E7792B3D680AE25C6F4DBBA09114D165"
 	},
-	{
-		.addr4 = "193.124.186.205",
-		.addr6 = "2a02:f680:1:1100::542a",
-		.udp_port = 5228,
-		.tcp_port = 3389,
-		.idstr = "9906D65F2A4751068A59D30505C5FC8AE1A95E0843AE9372EAFA3BAB6AC16C2C"
-	},
 	{
 		.addr4 = "80.87.193.193",
 		.addr6 = "2a01:230:2:6::46a8",
 		.udp_port = 33445,
-		.tcp_port = 3389,
+		.tcp_port = 33445,
 		.idstr = "B38255EE4B054924F6D79A5E6E5889EC94B6ADF6FE9906F97A3D01E3D083223A"
 	},
 	{
 		.addr4 = "initramfs.io",
-		.addr6 = NULL,
+		.addr6 = "tox.initramfs.io",
 		.udp_port = 33445,
-		.tcp_port = 33445,
+		.tcp_port = 3389,
 		.idstr = "3F0A45A268367C1BEA652F258C85F4A66DA76BCAA667A49E770BCC4917AB6A25"
 	},
 	{
@@ -195,16 +174,23 @@ static struct node nodes[] = {
 		.tcp_port = 33445,
 		.idstr = "C7D284129E83877D63591F14B3F658D77FF9BA9BA7293AEB2BDFBFE1A803AF47"
 	},
+	{
+		.addr4 = "46.229.52.198",
+		.addr6 = NULL,
+		.udp_port = 33445,
+		.tcp_port = 33445,
+		.idstr = "813C8F4187833EF0655B10F7752141A352248462A567529A38B6BBF73E979307"
+	},
 	{
 		.addr4 = "node.tox.ngc.network",
 		.addr6 = "node.tox.ngc.network",
 		.udp_port = 33445,
-		.tcp_port = 3389,
+		.tcp_port = 33445,
 		.idstr = "A856243058D1DE633379508ADCAFCF944E40E1672FF402750EF712E30C42012A"
 	},
 	{
-		.addr4 = "144.217.86.39",
-		.addr6 = "fe80::f816:3eff:fe61:92e3",
+		.addr4 = "149.56.140.5",
+		.addr6 = "2607:5300:0201:3100:0000:0000:0000:3ec2",
 		.udp_port = 33445,
 		.tcp_port = 3389,
 		.idstr = "7E5668E0EE09E19F320AD47902419331FFEE147BB3606769CFBE921A2A2FD34C"
@@ -213,70 +199,119 @@ static struct node nodes[] = {
 		.addr4 = "185.14.30.213",
 		.addr6 = "2a00:1ca8:a7::e8b",
 		.udp_port = 443,
-		.tcp_port = 3389,
+		.tcp_port = 443,
 		.idstr = "2555763C8C460495B14157D234DD56B86300A2395554BCAE4621AC345B8C1B1B"
 	},
 	{
-		.addr4 = "85.21.144.224",
+		.addr4 = "tox.natalenko.name",
+		.addr6 = "tox.natalenko.name",
+		.udp_port = 33445,
+		.tcp_port = 33445,
+		.idstr = "1CB6EBFD9D85448FA70D3CAE1220B76BF6FCE911B46ACDCF88054C190589650B"
+	},
+	{
+		.addr4 = "136.243.141.187",
+		.addr6 = "2a01:4f8:212:2459::a:1337",
+		.udp_port = 443,
+		.tcp_port = 33445,
+		.idstr = "6EE1FADE9F55CC7938234CC07C864081FC606D8FE7B751EDA217F268F1078A39"
+	},
+	{
+		.addr4 = "tox.abilinski.com",
 		.addr6 = NULL,
 		.udp_port = 33445,
 		.tcp_port = 33445,
-		.idstr = "8F738BBC8FA9394670BCAB146C67A507B9907C8E564E28C2B59BEBB2FF68711B"
+		.idstr = "0E9D7FEE2AA4B42A4C18FE81C038E32FFD8D907AAA7896F05AA76C8D31A20065"
 	},
 	{
-		.addr4 = "tox.natalenko.name",
-		.addr6 = "tox.natalenko.name",
+		.addr4 = "95.215.46.114",
+		.addr6 = "2a02:7aa0:1619::bdbd:17b8",
+		.udp_port = 33445,
+		.tcp_port = 3389,
+		.idstr = "5823FB947FF24CF83DDFAC3F3BAA18F96EA2018B16CC08429CB97FA502F40C23"
+	},
+	{
+		.addr4 = "m.loskiq.it",
+		.addr6 = NULL,
 		.udp_port = 33445,
 		.tcp_port = 33445,
-		.idstr = "1CB6EBFD9D85448FA70D3CAE1220B76BF6FCE911B46ACDCF88054C190589650B"
+		.idstr = "88124F3C18C6CFA8778B7679B7329A333616BD27A4DFB562D476681315CF143D"
 	},
 	{
-		.addr4 = "37.187.122.30",
+		.addr4 = "tmux.ru",
 		.addr6 = NULL,
 		.udp_port = 33445,
 		.tcp_port = 33445,
-		.idstr = "BEB71F97ED9C99C04B8489BB75579EB4DC6AB6F441B603D63533122F1858B51D"
+		.idstr = "7467AFA626D3246343170B309BA5BDC975DF3924FC9D7A5917FBFA9F5CD5CD38"
 	},
 	{
-		.addr4 = "completelyunoriginal.moe",
-		.addr6 = "completelyunoriginal.moe",
+		.addr4 = "37.48.122.22",
+		.addr6 = "2001:1af8:4700:a115:6::b",
+		.udp_port = 33445,
+		.tcp_port = 3389,
+		.idstr = "1B5A8AB25FFFB66620A531C4646B47F0F32B74C547B30AF8BD8266CA50A3AB59"
+	},
+	{
+		.addr4 = "tox.novg.net",
+		.addr6 = NULL,
+		.udp_port = 33445,
+		.tcp_port = 3389,
+		.idstr = "D527E5847F8330D628DAB1814F0A422F6DC9D0A300E6C357634EE2DA88C35463"
+	},
+	{
+		.addr4 = "t0x-node1.weba.ru",
+		.addr6 = NULL,
 		.udp_port = 33445,
 		.tcp_port = 33445,
-		.idstr = "FBC7DED0B0B662D81094D91CC312D6CDF12A7B16C7FFB93817143116B510C13E"
+		.idstr = "5A59705F86B9FC0671FDF72ED9BB5E55015FF20B349985543DDD4B0656CA1C63"
 	},
 	{
-		.addr4 = "136.243.141.187",
-		.addr6 = "2a01:4f8:212:2459::a:1337",
-		.udp_port = 443,
-		.tcp_port = 443,
-		.idstr = "6EE1FADE9F55CC7938234CC07C864081FC606D8FE7B751EDA217F268F1078A39"
+		.addr4 = "109.195.99.39",
+		.addr6 = NULL,
+		.udp_port = 33445,
+		.tcp_port = 33445,
+		.idstr = "EF937F61B4979B60BBF306752D8F32029A2A05CD2615B2E9FBFFEADD8E7D5032"
 	},
 	{
-		.addr4 = "tox.abilinski.com",
+		.addr4 = "79.140.30.52",
 		.addr6 = NULL,
 		.udp_port = 33445,
 		.tcp_port = 33445,
-		.idstr = "0E9D7FEE2AA4B42A4C18FE81C038E32FFD8D907AAA7896F05AA76C8D31A20065"
+		.idstr = "FFAC871E85B1E1487F87AE7C76726AE0E60318A85F6A1669E04C47EB8DC7C72D"
 	},
 	{
-		.addr4 = "95.215.46.114",
-		.addr6 = "2a02:7aa0:1619::bdbd:17b8",
+		.addr4 = "94.41.167.70",
+		.addr6 = NULL,
 		.udp_port = 33445,
 		.tcp_port = 33445,
-		.idstr = "5823FB947FF24CF83DDFAC3F3BAA18F96EA2018B16CC08429CB97FA502F40C23"
+		.idstr = "E519B2C1098999B60190012C7B53E8C43A73C535721036CD9DEC7CCA06741A7D"
 	},
 	{
-		.addr4 = "78.156.250.2",
+		.addr4 = "104.223.122.204",
 		.addr6 = NULL,
 		.udp_port = 33445,
+		.tcp_port = 3389,
+		.idstr = "3925752E43BF2F8EB4E12B0E9414311064FF2D76707DC7D5D2CCB43F75081F6B"
+	},
+	{
+		.addr4 = "77.55.211.53",
+		.addr6 = NULL,
+		.udp_port = 53,
 		.tcp_port = 443,
-		.idstr = "88124F3C18C6CFA8778B7679B7329A333616BD27A4DFB562D476681315CF143D"
+		.idstr = "B9D109CC820C69A5D97A4A1A15708107C6BA85C13BC6188CC809D374AFF18E63"
+	},
+	{
+		.addr4 = "boseburo.ddns.net",
+		.addr6 = NULL,
+		.udp_port = 33445,
+		.tcp_port = 33445,
+		.idstr = "AF3FC9FC3D121E82E362B4FA84A53E63F58C11C2BA61D988855289B8CABC9B18"
 	},
 	{
 		.addr4 = "46.101.197.175",
 		.addr6 = "2a03:b0c0:3:d0::ac:5001",
 		.udp_port = 443,
-		.tcp_port = 3389,
+		.tcp_port = 33445,
 		.idstr = "CD133B521159541FB1D326DE9850F5E56A6C724B5B8E5EB5CD8D950408E95707"
 	}
 };
